src/pages/Home.jsx

```diff
@@ -89,6 +99,18 @@ export default function Home() {
           line_colour={1}
           className="card"
         />
+
+        <DashboardGraph
+          title="Altitude"
```

Comment:
You're using acceleration data but calling the graph 'Altitude'. Could you change this to acceleration?